modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,262 @@
+* `Microsoft.Resources/subscriptions/resourcegroups/write`
+====
+
+.Required permissions for creating resource tags
+[%collapsible]
+====
+* `Microsoft.Resources/tags/write`
+====
+
+.Required permissions for creating storage resources
+[%collapsible]
+====
+* `Microsoft.Storage/storageAccounts/blobServices/read`
+* `Microsoft.Storage/storageAccounts/blobServices/containers/write`
+* `Microsoft.Storage/storageAccounts/fileServices/read`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/read`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/write`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/delete`
+* `Microsoft.Storage/storageAccounts/listKeys/action`
+* `Microsoft.Storage/storageAccounts/read`
+* `Microsoft.Storage/storageAccounts/write`
+====
+
+.Optional permissions for creating marketplace virtual machine resources
+[%collapsible]
+====
+* `Microsoft.MarketplaceOrdering/offertypes/publishers/offers/plans/agreements/read`
+* `Microsoft.MarketplaceOrdering/offertypes/publishers/offers/plans/agreements/write`
+====
+
+.Optional permissions for creating compute resources
+[%collapsible]
+====
+* `Microsoft.Compute/images/read`
+* `Microsoft.Compute/images/write`
+* `Microsoft.Compute/availabilitySets/write`
+* `Microsoft.Compute/images/delete`
+====
+
+.Optional permissions for enabling user-defined disk encryption set
+[%collapsible]
+====
+* `Microsoft.Compute/diskEncryptionSets/read`
+* `Microsoft.Compute/diskEncryptionSets/write`
+* `Microsoft.Compute/diskEncryptionSets/delete`
+* `Microsoft.KeyVault/vaults/read`
+* `Microsoft.KeyVault/vaults/write`
+* `Microsoft.KeyVault/vaults/delete`
+* `Microsoft.KeyVault/vaults/deploy/action`
+* `Microsoft.KeyVault/vaults/keys/read`
+* `Microsoft.KeyVault/vaults/keys/write`
+* `Microsoft.Features/providers/features/register/action`
+====
+
+.Optional permissions for installing a private cluster with Azure Network Address Translation (NAT)
+[%collapsible]
+====
+* `Microsoft.Network/natGateways/join/action`
+* `Microsoft.Network/natGateways/read`
+* `Microsoft.Network/natGateways/write`
+====
+
+.Optional permissions for installing a private cluster with Azure firewall
+[%collapsible]
+====
+* `Microsoft.Network/azureFirewalls/applicationRuleCollections/write`
+* `Microsoft.Network/azureFirewalls/read`
+* `Microsoft.Network/azureFirewalls/write`
+* `Microsoft.Network/routeTables/join/action`
+* `Microsoft.Network/routeTables/read`
+* `Microsoft.Network/routeTables/routes/read`
+* `Microsoft.Network/routeTables/routes/write`
+* `Microsoft.Network/routeTables/write`
+* `Microsoft.Network/virtualNetworks/peer/action`
+* `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/read`
+* `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write`
+====
+
+Following are the permissions required for deleting the {product-title} cluster on Azure public cloud. You can use the same permissions to delete the private {product-title} cluster on Azure.
```

Comment:
Since network DNS zones related permissions are not required for private cluster. Maybe we can reword here.

You can use the same permissions to delete the private {product-title} cluster on Azure.
=>
You can use following permissions but remove network DNS zone related permissions to delete the private {product-title} cluster on Azure.

@xenolinux @chiragkyal any suggestion?

Reply:
Instead can we add a note somewhere to skip those permissions when deleting a private cluster?

Reply:
If it's possible to add a NOTE inside the collapsible, then that would be great.

Reply:
@chiragkyal @jinyunma Added two notes for creating and deleting the private {product-title} cluster on Azure. 
https://github.com/openshift/openshift-docs/pull/56274/files#diff-c82de66f7c129811e1de419d2c8b04c6cf8429d036db2acffc36f43e606fdcbaR237

https://github.com/openshift/openshift-docs/pull/56274/files#diff-c82de66f7c129811e1de419d2c8b04c6cf8429d036db2acffc36f43e606fdcbaR94

Reply:
Added the note outside of the collapsible block